sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateFormatter.scala

```diff
@@ -117,7 +117,16 @@ class LegacySimpleDateFormatter(pattern: String, locale: Locale) extends LegacyD
 object DateFormatter {
   import LegacyDateFormats._
 
-  val defaultLocale: Locale = Locale.US
+  /**
+   * This is change from Locale.US to GB, because:
+   * The first day-of-week varies by culture.
+   * For example, the US uses Sunday, while the United Kingdom and the ISO-8601 standard use Monday.
```

Comment:
is this the only difference between US and en-GB?

Reply:
It seems that we don't need care about the other differences, e.g. currency symbol
FYI,
http://www.localeplanet.com/java/en-GB/index.html and http://www.localeplanet.com/java/en-US/index.html

the timeZone is not the same, but it's a separate field we don't get it from Locale right?

Reply:
is there any localized timezone related pattern letter?

Reply:
We have `O`, `OOOO` and `ZZZZ` that are localized, but there are decided by the `zoneID`(spark.sql.session.timeZone), not related to Locate here.